aten/src/ATen/native/Resize.cpp

```diff
@@ -43,7 +34,7 @@ bool resize_output(const Tensor& output, IntArrayRef shape) {
     //
     // TODO(#61485): functorch wrapped tensors should not go through the
     // fast path. This is a hack, longer term solutions are in the issue
-    if (output.is_cpu() && !is_functorch_wrapped_tensor(output)) {
+    if (output.is_cpu() && !isFuncTorchWrappedTensor(output)) {
```

Comment:
Why can't this be a subclass like check?

Reply:
It can